Resources/doc/reference/installation.rst

```diff
@@ -101,3 +101,9 @@ Installation
                     mappings:
                         ApplicationSonataClassificationBundle: ~
                         # ...
+
+* Update the schema:
+
+.. code-block:: bash
+
+    php bin/console doctrine:schema:update --force
```

Comment:
:-1: for that.

`doctrine:schema:update` is not a command to use on production for example, you should use migrations.